examples/java/src/main/java/org/apache/beam/examples/complete/TopWikipediaSessions.java

```diff
@@ -193,11 +208,11 @@ public static void main(String[] args) {
 
     double samplingThreshold = 0.1;
 
-    p.apply(TextIO.Read
-        .from(options.getInput())
-        .withCoder(TableRowJsonCoder.of()))
-     .apply(new ComputeTopSessions(samplingThreshold))
-     .apply("Write", TextIO.Write.withoutSharding().to(options.getOutput()));
+    p.apply(TextIO.Read.from(options.getInput()))
+        .apply(MapElements.via(new ParseTableRowJson())
+            .withOutputType(TypeDescriptor.of(TableRow.class)))
```

Comment:
I believe you should not need the `withOutputType` here if you simply make `ParseTableRowJson` above a `SimpleFunction`.